### ISSUE TITLE
Package fails to install on systems without `pkg-config`

### ISSUE THREAD

**Issue by cesarkawakami** [python-lz4/python-lz4#1]
On systems without `pkg-config`, `pip install lz4` fails with the following output:

```
Collecting lz4 (from ***)
  Downloading lz4-0.8.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-D9kUKL/lz4/setup.py", line 19, in <module>
        liblz4_found = subprocess.call(cmd) == 0
      File "/usr/lib/python2.7/subprocess.py", line 522, in call
        return Popen(*popenargs, **kwargs).wait()
      File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
        errread, errwrite)
      File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
        raise child_exception
    OSError: [Errno 2] No such file or directory
```

I've investigated this a little bit, and it seems like the following chunk of code on the [setup.py file][setuppy] isn't handling all exceptional cases correctly:

```
# Check to see if we have a lz4 library installed on the system and
# use it if so. If not, we'll use the bundled library. If lz4 is
# installed it will have a pkg-config file, so we'll use pkg-config to
# check for existence of the library.
pkg_config_exe = os.environ.get('PKG_CONFIG', None) or 'pkg-config'
cmd = '{0} --exists liblz4'.format(pkg_config_exe).split()
liblz4_found = subprocess.call(cmd) == 0
```

In this specific case, the `pkg-config` binary doesn't exist on the system, making `subprocess.call()` fail with an `OSError` exception instead of a non-zero return code, and this exception isn't being handled by the code, making the installation fail.

This can be pretty common on minimal installations. In this particular situation, I'm running pip inside an ubuntu docker container, and it seems like the default docker ubuntu image doesn't contain `pkg-config` by default:

```
$ sudo docker run --rm -it ubuntu bash
root@f8481c78cd7b:/# pkg-config
bash: pkg-config: command not found
```

In my case I think I'll try to solve this by installing the `pkg-config` package, but since this might trip other people I've also decide to open this issue.

[setuppy]: https://github.com/python-lz4/python-lz4/blob/7dc8ce69a98fb63a9a6a92a25179d942b6082dfd/setup.py#L19

**Comment by ZhukovAlexander** [comment]
+1 (installing `pkg-config` solved the issue)

**Comment by jonathanunderwood** [comment]
Ah yes. This should be fixed with commit 926b7d9081ff3ba743465d852ad47d8f6fcba42a
